### PR TITLE
Add docs for some less common RBAC configuration scenarios

### PR DESCRIPTION
**Pull request by GrahamGoudeau**

### CODE REVIEW

File: docs/content/advanced_configuration/rbac_configuration.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+You may want to prevent Gloo's RBAC resources from being created at install time. An example of this use
+case is if you are installing to a cluster where you have insufficient permissions to create privileged
+resources like RBAC roles and role-bindings.
+
+Gloo's RBAC resources will not be installed if the Helm flag `global.glooRbac.create` is set to `false`.
+
+Our Grafana and Prometheus subcharts will still attempt to create their own RBAC resources. If you would
+like to also prevent those resources from being creeated, you must also set the flags `grafana.rbac.create`
+and `prometheus.rbac.create` to `false`.
```
Comment (kdorosh) [inline docs/content/advanced_configuration/rbac_configuration.md]: worth mentioning that this is enterprise-only

File: docs/content/advanced_configuration/rbac_configuration.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Conditional Installation
+
+You may want to prevent Gloo's RBAC resources from being created at install time. An example of this use
+case is if you are installing to a cluster where you have insufficient permissions to create privileged
+resources like RBAC roles and role-bindings.
+
+Gloo's RBAC resources will not be installed if the Helm flag `global.glooRbac.create` is set to `false`.
+
+Our Grafana and Prometheus subcharts will still attempt to create their own RBAC resources. If you would
+like to also prevent those resources from being creeated, you must also set the flags `grafana.rbac.create`
```
Comment (marcogschmidt) [inline docs/content/advanced_configuration/rbac_configuration.md]:
```suggestion
like to also prevent those resources from being created, you must also set the flags `grafana.rbac.create`
```

File: docs/content/advanced_configuration/rbac_configuration.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## Uniquely Identifying Cluster-Scoped Resources
+
+By default, Gloo's RBAC resources are created as cluster-scoped. This may present a problem if, for example,
+you upgrade Gloo by performing a 
+[blue-green deployment](https://blog.christianposta.com/deploy/blue-green-deployments-a-b-testing-and-canary-releases/)
+and deploying Gloo to two namespaces simultaneously within the same cluster. The identically-named 
+cluster-scoped roles and role-bindings will conflict with each other and will cause an error.
+
+To resolve this, you may set the Helm flag `global.glooRbac.roleSuffix` to some string of your choosing.
+That will cause the string `"-$roleSuffix"` to be appended to the roles' and role-bindings' names, where
```
Comment (marcogschmidt) [inline docs/content/advanced_configuration/rbac_configuration.md]: Nit, does this make sense?
```suggestion
That will cause the string `"-$roleSuffix"` to be appended to the role and role-binding names, where
```

File: docs/content/advanced_configuration/rbac_configuration.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+`$roleSuffix` is the string you passed to the Helm flag. For example, setting `global.glooRbac.roleSuffix`
+to the string `"blue-deployment"` will cause the ClusterRole normally named `gloo-resource-reader` to
```
Comment (marcogschmidt) [inline docs/content/advanced_configuration/rbac_configuration.md]: Another nit: you use both the "role" / "role-binding" and "ClusterRole" forms in this page, can we use a single convention? (I prefer the k8s names, e.g.  ClusterRole).

**Pull request merged**